### PR TITLE
klassy: init at 4.3.breeze5.27.5

### PR DESCRIPTION
**Pull request by ghost**
###### Description of changes

Klassy (formerly ClassiK/ClassikStyles) is a highly customizable binary Window Decoration and Application Style plugin for recent versions of the KDE Plasma desktop. [Initially taking inspiration from the iconography of KDE 1](https://forum.kde.org/viewtopic.php?f=285&t=138602?raw=true), the Klassy defaults are an attempt to create a usable and appealing look for the modern Plasma desktop.

### CODE REVIEW

**Review by geri1701 — COMMENTED** [review]
`added myself as a maintainer` should be made a standalone commit so you have two commits afterwards.

**Comment by HikariNee** [comment]
Is this alright?

**Comment by geri1701** [comment]
> Is this alright?

Yes, that looks good, but we should have two commits here, so delete #3916c67 and force push again.

**Comment by HikariNee** [comment]
Done.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -6274,6 +6274,14 @@
     github = "higebu";
     githubId = 733288;
   };
+
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:6277]: I don't think there should be new lines here looking at the other entries

File: pkgs/data/themes/klassy/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  meta = with lib; {
+    description = "Klassy (formerly ClassiK/ClassikStyles) is a highly customizable binary Window Decoration and Application Style plugin for recent versions of the KDE Plasma desktop";
+    homepage = "https://github.com/paulmcauley/klassy";
+    license = licenses.gpl2;
+    maintainers = [maintainers.hikari];
```
Comment (SuperSandro2000) [inline pkgs/data/themes/klassy/default.nix:39]:
```suggestion
    maintainers = [ maintainers.hikari ];
```
nit

File: pkgs/data/themes/klassy/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}:
+
+mkDerivation rec {
+  pname = "klassy";
+  version = "4.1";
+
+  src = fetchFromGitHub {
+    owner = "paulmcauley";
+    repo = "klassy";
+    rev = "972edd08184b3a416b166053a1b1a3d042d33d92";
```
Comment (SuperSandro2000) [inline pkgs/data/themes/klassy/default.nix:19]:
```suggestion
  version = "4.1.breeze5.25.80";

  src = fetchFromGitHub {
    owner = "paulmcauley";
    repo = "klassy";
    rev = version;
```
thats the actual version according to cmake

File: pkgs/data/themes/klassy/default.nix [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+, kdecoration
+, qt5
+, cmake
+, extra-cmake-modules
+, plasma-workspace
+, qtbase
+, fetchFromGitHub
+}:
+
+mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/data/themes/klassy/default.nix:12]:
```suggestion
stdenv.mkDerivation rec {
```

File: pkgs/data/themes/klassy/default.nix [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{mkDerivation
+, lib
```
Comment (SuperSandro2000) [inline pkgs/data/themes/klassy/default.nix:2]:
```suggestion
{ lib
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/data/themes/klassy/default.nix (hunk 5 above)
Comment (HikariNee) [reply pkgs/data/themes/klassy/default.nix:2]: Should I amend the commits or add these?

**Comment by HikariNee** [comment]
I seem to have done something I cannot fix. Is it alright if I open a separate PR with your suggestions?

**Comment by HikariNee** [comment]
Status?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
You are missing a top-level entry. Please add one.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**